### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by alowe4**

### CODE REVIEW

**Comment by alowe4** [comment]
Pull Request title includes "Assignment 1" in the title ✔
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔

**Comment by alowe4** [comment]
done with no partner

**Comment by Howard404** [comment]
I will review this

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ public static void Main(string[] args)
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            return ((double)numberCorrectAnswers / numberOfQuestions * 100) + "%";
```
Comment (Howard404) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:27]: GetPercentCorrect() method should return an integer value. Try storing the casted double operation to a variable and return the variable as a integer type

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 2]
```diff
@@ -89,6 +89,7 @@ public static Question[] LoadQuestions(string filePath)
                 question.Answers[1] = answer2;
                 question.Answers[2] = answer3;
                 question.CorrectAnswerIndex = correctAnswerIndex;
+                questions[i] = question;
```
Comment (Howard404) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:92]: Great work

**Review by Howard404 — COMMENTED** [review]
Pull Request title includes "Assignment 1" in the title ✔
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔

**Comment by mmwoodfo** [comment]
Make sure to find yourself a partner in the future!

**Pull request closed without merging**